activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Stop using `lower()` for case-insensitive queries on `citext` columns
```

Comment:
I also should have mentioned, that here `lower()` in the CHANGELOG also needs to be replaced with `LOWER()`.